save-cloud-common/src/jvmMain/kotlin/com/saveourtool/save/utils/FileUtils.kt

```diff
@@ -56,6 +58,17 @@ fun Flux<DataBuffer>.writeTo(target: Path): Mono<Path> =
             .map { DataBufferUtils.release(it) }
             .then(Mono.just(target))
 
+/**
+ * @param target path to file to where a content from receiver will be written
+ * @return [Mono] with [target]
+ */
+fun Flux<ByteBuffer>.writeToFile(target: Path): Mono<Path> = map { byteBuffer ->
+    target.outputStream(StandardOpenOption.CREATE, StandardOpenOption.APPEND).use { os ->
```

Comment:
- What happens if the target file already exists?
- What happens if the function is invoked twice in arow with the same argument?

Either make sure the existing file is **truncated**, or **document** what happens if the file exists (e.g.: an `IOException` is thrown).

Reply:
Improved KDoc